Meta tags

JimenaEB opened the pull request:
Cambios de optimización SEO

Commit:
Cambia meta tags
Por optimización SEO las etiquetas meta y link deben cerrarse sin usar </ #nombre--tag>

Guzmangm commented:
¿Lo podemos hacer con tags para mantener la misma línea que https://github.com/the-cocktail/iberostar-web/pull/770/files?

JimenaEB commented:
Sí!!! Perdona, pero fueron mis primeros pinitos con ruby y aun no sabía formarlos con tag, voy a ello 😉

Commit:
Cambia etiquetas hechas como String por tag
Cambia las etiquetas que hay que escapar formadas como un String por etiquetas formadas con tag()
